src/earn/EarnCard.tsx

```diff
@@ -19,8 +20,8 @@ export function EarnCardDiscover({ depositTokenId, poolTokenId }: Props) {
   const poolToken = useTokenInfo(poolTokenId)
```

Comment:
I'm a bit confused what this would be in the multi-pool case. Should the props to this function be refactored? It seems like we would need to check if any of the pool tokens have a balance to decide to show `EarnActivePools`.

Reply:
For multi pool, this would just be based on the `getEarnPools` output. The tokenId props would just be for the earn MVP. We can remove those when we remove MVP specific code.